docs/mobileAppSelfDefenceOverview.adoc

```diff
@@ -248,16 +248,7 @@ Detecting if the application is built in debug mode.
 .Code
 [source,javascript,linenums,indent=0]
 ....
-include::https://raw.githubusercontent.com/feedhenry/mobile-security-cordova-template/master/src/pages/deviceTrust/deviceTrust.ts[tags=detectDebug]
-....
-
-==== Detecting Latest OS
-Detecting if the application is running the latest version of Android or iOS.
-
-.Code
-[source,javascript,linenums,indent=0]
-....
-include::https://raw.githubusercontent.com/feedhenry/mobile-security-cordova-template/master/src/pages/deviceTrust/deviceTrust.ts[tags=detectLatestOS]
+include::https://raw.githubusercontent.com/aerogear/cordova-showcase-template/master/src/pages/security/deviceTrust/deviceTrust.ts[tags=detectDebug]
```

Comment:
are the tags still there in the showcase app?

Reply:
yes still there